packages/astro/src/internal/create-clerk-instance.ts

```diff
@@ -53,6 +54,11 @@ async function createClerkInstanceInternal(options?: AstroClerkCreateInstancePar
     .load(initOptions)
     .then(() => {
       $csrState.setKey('isLoaded', true);
+      // Notify subscribers that $clerkStore has been loaded.
+      // We're doing this because nanostores uses `===` for equality
+      // and just by setting the value to `window.Clerk` again won't trigger an update.
+      // We notify only once as this store is for advanced users.
+      $clerkStore.notify();
```

Comment:
`notify` will let the `atom` know internally that `window.Clerk` has been updated